client/src/components/Visualiser/Controller/VisualiserController.tsx

```diff
@@ -119,17 +117,33 @@ const VisualiserController: FC<Props> = ({
           <Stack direction="column">
             <Stack direction="row" sx={{ height: '32px' }}>
               <TimeIcon className={styles.sliderIcon} sx={{ fill: theme.palette.text.primary }} />
-              <div className="col">
-                <input
-                  type="range"
-                  id="timelineSlider"
-                  name="volume"
-                  min="0"
-                  max="100"
-                  defaultValue="0"
-                  step="0.01"
-                />
-              </div>
+              <input
```

Comment:
The sliders are different.

Reply:
I assume you mean one is mui while the other is an html input element. The reason it's like that is because updating the animationProgress state many times per second was slowing down the visualiser quite a bit